react/features/calendar-sync/web/googleCalendar.js

```diff
@@ -0,0 +1,80 @@
+/* @flow */
+
+import {
+    getCalendarEntries,
+    googleApi,
+    loadGoogleAPI,
+    signIn,
+    signOut,
+    updateProfile
+} from '../../google-api';
+import { CALENDAR_TYPE } from '../constants';
+
+declare var config: Object;
+
+/**
+ * A stateless object that implements the expected interface for interacting
+ * Google authentication in order to get calendar data.
+ *
+ * @type {Object}
```

Comment:
Probably the doc should mention how pretty much this object is an aggregation of actions that need to be dispatched.